python/ray/_raylet.pyx

```diff
@@ -1321,7 +1322,11 @@ cdef class CoreWorker:
         CCoreWorkerProcess.GetCoreWorker().RemoveActorHandleReference(
             c_actor_id)
 
-    cdef make_actor_handle(self, const CActorHandle *c_actor_handle):
+    cdef make_actor_handle(self, ActorHandleSharedPtr actor_handle_shared_ptr):
```

Comment:
this function should probably take a shared pointer ref to avoid incrementing the shared ptr

Reply:
Cython doesn't support reference. Copying a `shared_ptr` is cheap. So I think it's fine.